bson/src/main/org/bson/json/JsonString.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2008-present MongoDB, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.bson.json;
+
+/**
+ * A wrapper class that holds a JSON string. This class makes decoding straight
+ * to JSON easy.
+ *
+ * @since 4.2
+ */
+public class JsonString {
+    private final String json;
+
+    /**
+     * Constructs a new instance with the given JSON string
+     *
+     * @param json the JSON string
+     */
+    public JsonString(final String json) {
```

Comment:
What happens if a user tries to do:

```
MongoCollection<JsonString> collection.insert(new JsonString("['a', 'b', 'c']"));
```

Do you think there should be any validation on the String?

Reply:
Now I'm wondering if we should call this class something different.  Since `[1, 2, 3]` (or even `1`?) is valid JSON, should this class be called `JsonObjectString` instead, if indeed it will forever be restricted to JSON objects (see https://www.json.org/json-en.html)?

Even if we don't change the name, if we are committing to only supporting JSON objects, then this class can and should implement `org.bson.conversions.Bson`.  That way, it can be used for query filters, etc.

Alternatively, we could allow any JSON string, not just JSON objects, though currently BsonReader.pipe currently only supports documents

Reply:
Done (renamed to `JsonObject` and implements `Bson`